test/test_refaddr_bitness.py

```diff
@@ -0,0 +1,49 @@
+#------------------------------------------------------------------------------
+# elftools tests
+#
+# Seva Alekseyev (sevaa@sprynet.com)
+# This code is in the public domain
+#------------------------------------------------------------------------------
+
+import unittest
+import os, sys, io
+
+# sys.path.insert(1, os.getcwd())
+
+from elftools.dwarf.dwarfinfo import DWARFInfo, DebugSectionDescriptor, DwarfConfig
+
+class TestRefAddrOnDWARFv2With64BitTarget(unittest.TestCase):
```

Comment:
Please add a substantial comment explaining what this test is testing/doing, and how.

Reply:
Commented